### ISSUE TITLE
No documentation of res.jsonx() - should there be?

### ISSUE THREAD

**Issue by chrisfrancis27** [balderdashy/sails-docs#413]
I noticed a few usages of `return res.jsonx(data)` throughout parts of the Sails blueprint APIs. It seems like a fairly useful wrapper around JSON/JSON-P response handling. Was it intended to be used externally? If so, happy to submit a pull request with some extra doc info.

**Comment by sgress454** [comment]
Hi @chrisfrancis27, and thanks!  Docs around `jsonx` would be great.

**Comment by tjwebb** [comment]
So, did this ever happen?

**Comment by smirnovshina** [comment]
Reading docs now, still no `res.jsonx()` description.

**Comment by mikermcneil** [comment]
If someone submits a PR which sets up a first pass at this, happy to add to it and get it merged in. This method is in userland code in generated response files; but was not intended for direct use other places in your app. (That said, we should have a reference page on it that explains that in a note)

**Comment by sailsbot** [comment]
Thanks for posting, @chrisfrancis27.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
